src/style.css

```
@@ -0,0 +1,119 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Avenir-Book", sans-serif;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.large-form {
+  position: relative;
+  width: 100%;
+  height: 70px;
+  margin-top: 20px;
+}
+
+.large-search-button,
+.large-input-line {
+  border: 0;
+  outline: 0;
+}
+
+.large-input-line {
+  width: 100%;
+  height: 100%;
+  padding: 24px 0 24px 62px;
+  font-family: inherit;
+  font-weight: inherit;
+  font-size: 16px;
+  color: ingerit;
```

Cортуєш правильно, тільки роби відступи між різними групами. Тоді їх простіше буде читати
```suggestion
  width: 100%;
  height: 100%;
  padding: 24px 0 24px 62px;
  
  font-family: inherit;
  font-weight: inherit;
  font-size: 16px;
  color: ingerit;
```